contracts/staking/src/contract.rs

```diff
@@ -52,52 +52,64 @@ pub fn instantiate(
     msg: InstantiateMsg,
 ) -> Result<Response, ContractError> {
     set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
-    let operators = validate_addresses(&msg.operators, OSMOSIS_ACCOUNT_PREFIX)?;
-    let validators = validate_addresses(&msg.validators, CELESTIA_VALIDATOR_PREFIX)?;
 
     // TODO: determine if info.sender is the admin or if we want to pass in with msg
     ADMIN.set(deps.branch(), Some(info.sender.clone()))?;
 
-    if msg.ibc_channel_id == "" {
-        return Err(ContractError::ConfigWrong {});
-    }
-
-    if msg.native_token_denom == "" {
-        return Err(ContractError::ConfigWrong {});
-    }
-
-    validate_address(
-        &msg.multisig_address_config
-            .reward_collector_address
-            .to_string(),
-        &CELESTIA_ACCOUNT_PREFIX,
-    )?;
-    validate_address(
-        &msg.multisig_address_config.staker_address.to_string(),
-        &CELESTIA_ACCOUNT_PREFIX,
-    )?;
+    // validations
+    let validators = validate_addresses(&msg.validators, CELESTIA_VALIDATOR_PREFIX)?;
+    assert!(
+        msg.liquid_stake_token_denom.len() > 3,
+        "liquid_stake_token_denom is required"
+    );
```

Comment:
Why do we need to assert if the length of `liquid_stake_token_denom` is greater than 3?

Reply:
That's just a sanity check. Any string with character would do. Want to change this to: 
```suggestion
    assert!(
        msg.liquid_stake_token_denom.len() > 0,
        "liquid_stake_token_denom is required"
    );
```